### DISCUSSION TITLE
The role of Code Reviewer in CFI?

### DISCUSSION THREAD

**Discussion by AdrianHammond** [finos/compliant-financial-infrastructure#323] [General]
During the RIWG we discussed if we should adopted code reviewer role, someone who is *trusted' but is not able to make the commitment to be a maintainer. What do others think? @abdullahgarcia @thinkl33t @eddie-knight

**Comment by eddie-knight** [comment]
Would like to see other's responses here, but my take on this is that it can be a good way to give authority without responsibility.

Currently we are reliant on maintainers for code reviews, but we also expect maintainers to have a high level of participation on the project. Adding a reviewer role will allow us to spread future work out to trusted individuals who are not able to make firm commitments to the project.

**Reply by AdrianHammond** [reply]
totally agree @eddie-knight

**Reply by abdullahgarcia** [reply]
I'm pretty sure there are already some guidelines within the Linux foundation on this matter. I'll look for them, but let's discuss this.
